pkgs/tools/misc/fastfetch/default.nix

```diff
@@ -88,8 +111,21 @@ stdenv.mkDerivation (finalAttrs: {
   ]);
 
   cmakeFlags = [
-    "-DCMAKE_INSTALL_SYSCONFDIR=${placeholder "out"}/etc"
-    "-DENABLE_SYSTEM_YYJSON=YES"
+    (lib.cmakeOptionType "filepath" "CMAKE_INSTALL_SYSCONFDIR" "${placeholder "out"}/etc")
+    (lib.cmakeBool "ENABLE_DIRECTX_HEADERS" false)
```

Comment:
Does fastfetch run on Windows? O.o

Reply:
The fastfetch page has a binary for Windows and it works :)